src/Paragon/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+export const ParagonContext = React.createContext({});
+
+const Paragon = ({ dependencies, children }) => {
+  const contextValue = useMemo(
```

Comment:
note: context values should always be memoized so the value passed into the context doesn't change it's reference on each re-render of the component.